Source/AFError.swift

```diff
@@ -459,12 +483,17 @@ extension AFError: LocalizedError {
             return reason.localizedDescription
         case .multipartEncodingFailed(let reason):
             return reason.localizedDescription
+        case .requestAdaptationFailed(let error):
+            return "Request adaption failed with error: \(error.localizedDescription)"
         case .responseValidationFailed(let reason):
             return reason.localizedDescription
         case .responseSerializationFailed(let reason):
             return reason.localizedDescription
         case .serverTrustEvaluationFailed:
             return "Server trust evaluation failed."
+        case .requestRetryFailed(let retryError, let originError):
```

Comment:
Should this be `orignalError` rather than `originError`? That seems like an awkward name and has connotations of "network origin" which may be confusing to some users.

Reply:
You bet. I went back and forth on `originalError` vs. `originError` and couldn't really decide.

Reply:
Renamed in f1a67df3 to `originalError`.